components/monitoring/grafana/development/kustomization.yaml

```diff
@@ -2,3 +2,10 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+patches:
+  - path: auto-assign-role-patch.yaml
+    target:
+      name: grafana-oauth
+      kind: Grafana
+
+
```

Comment:
Redundant line at the end of the file (you have 2 newlines)

Reply:
Fixed that, thanks!